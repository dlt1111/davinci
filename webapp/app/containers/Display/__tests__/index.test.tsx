import { cleanup } from '@testing-library/react'
import Display from '..'
import { renderWithReduxAndRouter } from 'app/test/utils'
import injectReducer from 'app/utils/injectReducer'
import displayReducer from '../reducer'
import viewReducer from 'containers/View/reducer'
import { compose } from 'redux'

afterEach(cleanup)

describe('Display component test', () => {
  it('should take a snapshot', () => {
    const withDisplayReducer = injectReducer({
      key: 'display',
      reducer: displayReducer
    })
    const withReducer = injectReducer({ key: 'view', reducer: viewReducer })
    const Comp = compose(withReducer, withDisplayReducer)(Display)
    const { asFragment } = renderWithReduxAndRouter(Comp)
    expect(asFragment()).toMatchSnapshot()
  })
})
