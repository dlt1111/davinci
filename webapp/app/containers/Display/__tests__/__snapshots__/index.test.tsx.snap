// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component test should take a snapshot 1`] = `<DocumentFragment />`;
